src/ResourceManager/Network/Commands.Network/Models/PSVirtualNetworkUsage.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Microsoft.Azure.Commands.Network.Models
+{
+    public partial class PSVirtualNetworkUsage
+    {
+        public string Id;
```

Comment:
@EvgenyAgafonchikov nit: for each of these properties, can we add the `get` and `set` methods to be consistent with the other model classes?

```cs
public string Id { get; set; }
```